### PR TITLE
feat: message(memory)

### PR DESCRIPTION
**Pull request by sunatchaa**

### CODE REVIEW

File: src/modules/message.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        return { message: "Failed to send message" };
+      }
+    },
+    { body: "sendMessageBody" }
+  )
+
+  .patch(
+    "/:id",
+    async ({ params, body, userId, set }) => {
+      const messageId = parseInt(params.id, 10);
```
Comment (khajornritdacha) [inline src/modules/message.ts:87]: Should return BAD request status where `params.id` is not a number.

File: src/modules/message.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        });
+
+        set.status = "OK";
+        return {
+          ...updatedMessage,
+          message_id: updatedMessage.message_id.toString(),
+          user_id: updatedMessage.user_id.toString(),
+        };
+      } catch (error) {
+        set.status = "Internal Server Error";
```
Comment (khajornritdacha) [inline src/modules/message.ts:126]: Should `console.error(error)` on the catch to log all errors that occurred in the system. The same goes for other try, catch statement

**Review by khajornritdacha — APPROVED** [review]
Just some minor changes. But everything else LGTM kub

**Review by MasterIceZ — APPROVED** [review]
lgtm

**Pull request merged**